### DISCUSSION TITLE
A way to generate report about all repositories in an organization (preferably as CSV)?

### DISCUSSION THREAD

**Discussion by ykartsev** [community/community#45168] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi dear GitHub Team/Community!

Is there a way to generate a report that would contain all repositories of an organization containing columns like "Type", "last updated", "# of members", "# of PRs", etc.  and sorted by a specified column? Would be awesome if there was also ability to export as CSV. The closest I found is "Repositories", but it has pages and does not allow to sort by other columns, as well as export to CSV.

Thank you in advance!

**Comment by seon22break** [comment, marked as answer]
Hi Yury , your suggestion is very interesting. But in the meantime you can test with the API.
